tests/validators/test_datetime.py

```diff
@@ -185,6 +186,14 @@ def test_tz_comparison():
         SchemaValidator({'type': 'datetime', 'gt': uk_3pm}).validate_python('2022-01-01T16:00:00+01:00')
 
 
+def test_tz_info_deepcopy():
+    output = SchemaValidator({'type': 'datetime'}).validate_python('2023-02-15T16:23:44.037Z')
+    c = copy.deepcopy(output)
```

Comment:
please can you test with `copy.copy` as well.

Reply:
Done. What do you think?